src/app/beer_garden/api/http/handlers/v1/user.py

```diff
@@ -147,3 +147,26 @@ def post(self):
         create_user(**user_data)
 
         self.set_status(201)
+
+
+class WhoAmIAPI(BaseHandler):
```

Comment:
I added this endpoint specifically to provide a simple thing to test against. It may or may not be worth keeping around in the end, but certainly having it doesn't hurt anything and until we're done with all of the auth changes it'll be useful to have.